src/js/pages/system/UnitsHealthDetail.js

```diff
@@ -138,49 +146,52 @@ class UnitsHealthDetail extends mixin(StoreMixin) {
       isLoadingUnit,
       isLoadingNodes
     } = this.state;
+    let content = null;
+    let unit = null;
 
     if (hasError) {
-      return this.getErrorNotice();
-    }
-
-    if (isLoadingUnit || isLoadingNodes) {
-      return this.getLoadingScreen();
+      content = this.getErrorNotice();
```

Comment:
Could we turn this into a separate function (e.g. `this.getContent()`) so we can `return` and keep them as simple `if`-statements?